tests/gradients/parameter_shift/test_parameter_shift_shot_vec.py

```diff
@@ -261,12 +261,13 @@ def circuit(params):
             return qml.probs([2, 3])
 
         params = np.array([0.5, 0.5, 0.5], requires_grad=True)
-        circuit.construct((params,), {})
```

Comment:
Are we removing the `construct` usage because it internally refers to the `_tape` variable, right?

Reply:
The `QNode` is constructed when you call `qml.workflow.construct_tape` 😄. Therefore, I'm just removing some redundant code.